packages/utils-helpers/src/isEmpty.ts

```diff
@@ -0,0 +1,24 @@
+//
+// Checks if the given item is empty
+//
+export default function isEmpty(item: any): boolean {
+    if (item == null) {
+        return true;
+    }
+
+    if (typeof item === 'number' || typeof item === 'boolean') {
+        return false;
+    }
```

Comment:
not to be nitpicking: and it doesnt matter in this case but `isEmpty(false)` will be `false` ... is this right?
otherwise 👍👍👍

```suggestion
    if (typeof item === 'number') {
        return false;
    }

    if (typeof item === 'boolean') {
        return item;
    }
```

Reply:
Correct `isEmpty(false)` and `isEmpty(true)` will be not empty, and every number is also not empty.
Maybe we can discuss if `isEmpty(NaN)` should be empty. Because `NaN` is a number but not really. So, this would be false.

Reply:
and hu there are 'functions' and so on - i must say i regret now that i said anything 😂

Reply:
i think as long as we dont try to reuse this rather specific `isEmpty` were all good to go and this can be merged ...

with specific i mean, in which language is `0` truthy and `''` falsy ? not in php or js at least ... so that makes this unpredictable ^^

Reply:
In my use cases I usually desire a differentiation if something is "set" or not. That means is it null/undefined or any other value.

But on the other hand the truthiness and falsiness of PHP also usually fulfils my demands except a check on an actual `false` value.

Reply:
Or at least we should rename it to its use case. Like `isOptionSet`

Reply:
i agree with @Sebobo,
when we figure out what the value can be and is when its not set - probably `null` or `undefined` we dont need to check for array lenght or object stuff - because the value will never be an object fx.